packages/stryker-elements/package.json

```diff
@@ -39,14 +39,25 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "storybook": "storybook dev -p 6006"
   },
   "peerDependencies": {
     "lit": "^2.7.6"
   },
   "devDependencies": {
+    "@chromatic-com/storybook": "^1.2.19",
+    "@storybook/addon-essentials": "^8.0.0",
+    "@storybook/addon-links": "^8.0.0",
+    "@storybook/blocks": "^8.0.0",
+    "@storybook/manager-api": "^8.0.0",
+    "@storybook/test": "^8.0.0",
+    "@storybook/theming": "^8.0.0",
+    "@storybook/web-components": "^8.0.0",
+    "@storybook/web-components-vite": "^8.0.0",
```

Comment:
Can we pin these dependencies? E.g.: `^8.0.0` -> `8.0.0` (or whatever is latest)

Reply:
8.0.0 is the latest version and has been pinned.